tests/stdlib/test_math.py

```diff
@@ -2,37 +2,81 @@
 
 from ..utils import TranspileTestCase
 
-
```

Comment:
According to flask8: ./tests/stdlib/test_math.py:5:1: E302 expected 2 blank lines, found 1.

(Tried to use suggestions, but Github didn't support empty line suggestions very well...)

Reply:
Fixed in latest commit